apps/hs/qua-server/src/Handler/QuaViewSettings.hs

```diff
@@ -22,6 +22,7 @@ getQuaViewEditorSettingsR
        , canDownloadGeometry    = True
        , canModifyStaticObjects = True
        , showHiddenProperties   = True
+       , showShareButton        = True
```

Comment:
Should be False, because there would be no good link and no notion of submission